builder-support/dockerfiles/Dockerfile.target.centos-6

```diff
@@ -10,6 +10,10 @@ RUN touch /var/lib/rpm/* && yum clean all
 RUN touch /var/lib/rpm/* && yum install -y --verbose epel-release centos-release-scl-rh && \
     yum install -y --nogpgcheck devtoolset-7-gcc-c++
 
+# Enable IUS Community Repository (for Python 3.6)
```

Comment:
the 3.4 in EPEL is too old?

Reply:
To be candid I didn't even try to use it, as it's far beyond EOL and adding the IUS repository wasn't difficult. You've already announced that Auth 4.4.x won't be supported on CentOS 6 anyway, and I assume that will be true for Recursor and DNSdist in their next releases too, so this target file will probably be removed soon :-)

Reply:
Works for me, and this should not have made IUS a runtime dep. Ok!